package-lock.json

```diff
@@ -1862,26 +2251,8 @@
       }
     },
     "node_modules/convex-helpers": {
-      "version": "0.1.27",
-      "resolved": "file:packages/convex-helpers",
-      "license": "MIT",
-      "peerDependencies": {
-        "convex": "^1.10.0",
-        "hono": "^4.0.5",
-        "react": "^17.0.2 || ^18.0.0",
-        "zod": "^3.22.4"
-      },
-      "peerDependenciesMeta": {
-        "hono": {
-          "optional": true
-        },
-        "react": {
-          "optional": true
-        },
-        "zod": {
-          "optional": true
-        }
-      }
+      "resolved": "packages/convex-helpers",
+      "link": true
```

Comment:
so this change keeps getting recommended by renovate but when I did the install like `npm i ./packages/convex-helpers` it un-does it, and doing `npm i` this way I think was not doing what I expected or I had trouble doing npm link or something. Do you know what this is about @thomasballinger - is this the "right" way to do it, vs. the `file:packages/convex-helpers` mode it's replacing? 
It would be nice to do it the way renovate keeps overriding, but if it's always fighting with the publish script it'll be annoying.

Reply:
Running `npm i ./package/convex-helpers` doesn't undo this for me and I still see live updating — so it seems to work.

I think we can stop using `npm link` and just rely on this "development" thing. I removed all npm links on my machine and still see these updating live. I'll remove the npm link stuff in this PR.

Reply:
Interestingly `"convex-helpers": "file:packages/convex-helpers"` always seems to result in a symlink at `node_modules/convex-helpers`, no `npm link` needed

Reply:
Oh what version of npm are you using? that could be the different behavior we see